homeassistant/components/tomorrowio/sensor.py

```diff
@@ -295,6 +295,7 @@ def convert_ppb_to_ugm3(molecular_weight: int | float) -> Callable[[float], floa
     TomorrowioSensorEntityDescription(
         key=TMRW_ATTR_UV_INDEX,
         name="UV Index",
+        state_class="measurement",
```

Comment:
Please use the sensor state class enum:
https://github.com/home-assistant/core/blob/8b4937f627126f5b6a16776e6def1a5070124641/homeassistant/components/sensor/const.py#L432